src/vs/editor/contrib/codeAction/browser/codeActionCommands.ts

```diff
@@ -222,7 +222,7 @@ export class OrganizeImportsAction extends EditorAction {
 		super({
 			id: organizeImportsCommandId,
 			label: nls.localize('organizeImports.label', "Organize Imports"),
-			alias: 'Organize Imports',
+			alias: 'Organize Imports, Optimize Imports',
```

Comment:
I'd leave the alias unchanged if possible. This may be confusing as it will result in the command being showing inconsistently depending on what UI language the editor is in